Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Pull base image
+FROM --platform=linux/x86_64 continuumio/miniconda3:23.10.0-1
```

Comment:
As a small suggestion, you might see if mambaforge can support your build, as it can make the environment build a lot faster. https://hub.docker.com/r/condaforge/miniforge3